plugins/genesys/fu-genesys-usbhub-fwbank.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2025 Adam.Chen <Adam.Chen@genesyslogic.com.tw>
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+enum FuGenesysUsbhubGl352530FwBank {
+	hub_bank1_addr = 0x0,
+	hub_bank2_addr = 0x10000,
+	hub_bank_capacity = 0x10000,
+	dev_bank1_addr = 0x20000,
+	dev_bank2_addr = 0x30000,
+	dev_bank_capacity = 0x8000,
+	pd_bank1_addr = 0x40000,
+	pd_bank2_addr = 0x60000,
+	pd_bank_capacity = 0x20000,
+	data_max_count = 0x80000,
```

Comment:
is this really an enum? I don't think it makes sense to create an enum just for the defines -- I'd much rather see this kind of data in the `.quirk` file anyway. e.g. add a quirk-only instance ID of `GENESYS\MODEL_GL352530` to the device, then add:

    [GENESYS\MODEL_WHATEVER]
    GenesysHubBank1Addr = 0x0

...and then catch the GenesysHubBank1Addr in `fu_genesys_usbhub_device_set_quirk_kv` and set it in the code there. I think the plugin would be a lot more maintainable that way than all those `if`s hardcoded.